auto_nag/scripts/topcrash_bad_severity.py

```diff
@@ -57,11 +57,13 @@ def get_bz_params(self, date):
                 "enhancement",
             ],
             "keywords": "topcrash",
-            "keywords_type": "allwords",
             "n1": 1,
             "f1": "longdesc",
             "o1": "casesubstring",
             "v1": "could you consider increasing the severity of this top-crash bug?",
+            "f2": "keywords",
+            "o2": "nowords",
+            "v2": "thunderbird",
```

Comment:
Won't this exclude cases where we have both "topcrash" and "topcrash-thunderbird" keywords?

Reply:
Yes, it will do. This is the intended behaviour.

Reply:
I have no problem doing that (just dropping L64-L66). But this will not cover the cases in #1597.

Reply:
As discussed on Matrix, https://bugzilla.mozilla.org/show_bug.cgi?id=1435961#c4 only has "topcrash-thunderbird" but no "topcrash", and same in https://bugzilla.mozilla.org/show_bug.cgi?id=1435343#c100 (after Jens fixed it to remove "topcrash" and only keep "topcrash-thunderbird").

Reply:
Covered in c69655002a82a8dd38ff0a181ae7b45a9a6079df